src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -161,7 +161,7 @@ public void traverse() {
         final Iterator<GATKRead> readIterator = getTransformedReadStream(countedFilter).iterator();
         // get the LIBS
         final LocusIteratorByState libs = new LocusIteratorByState(readIterator, getDownsamplingInfo(), keepUniqueReadListInLibs(), samples, header, includeDeletions(), includeNs());
-        Spliterator<AlignmentContext> iterator;
+        Iterator<AlignmentContext> iterator;
```

Comment:
Can this be final. Also did you were going to encapsulate the initialization in a private method?, perhaps it is in the next commit, disregard if that is the case.